### DISCUSSION TITLE
Is NC net still being called in that way?

### DISCUSSION THREAD

**Discussion by ribarraza** [xesscorp/skidl#147] [Q&A]
From documentation available here: https://devbisme.github.io/skidl/#no-connects , I understand that No-Connects net is called NC. However, when I try to use "NC" net, the tool crashes. To solve that I ended using NCNet(), but I am not sure if it is the same.

With that context, my questions are:

1) Are NC and NCNet() the same?
2) Is NC still in use?
3) If NC is not still in use, does that mean that documentation should be updated?

**Comment by devbisme** [comment]
`NC` and `NCNet()` are not the same. Everytime you call `NCNet()`, you'll get a different no-connect net. `NC` is a global no-connect net that is instantiated using `NCNet()`when SKiDL starts.

NC is still in use and its function is tested for every release. You could also use `default_circuit.NC` instead of `NC`.

What version of SKiDL are you using? What version of Python? What OS?

**Reply by ribarraza** [reply]
OOhh, now I see my error. I thought NC was a Skidl attribute, so I was getting a crash. Now I try using NC as a Net and it worked perfect.

Thank you for the answer.
